haproxy/README.md

```diff
@@ -115,7 +122,7 @@ spec:
 
 #### Using the stats endpoint
 
-**Note**: This configuration strategy is provided as a reference for legacy users. If you are setting up the integration for the first time, consider using the Prometheus-based strategy described in the previous section.
+<div class="alert alert-danger">This configuration strategy is provided as a reference for legacy users. If you are setting up the integration for the first time, consider using the Prometheus-based strategy described in the previous section.</div>
```

Comment:
Heads up that anything inside HTML tags gets stripped when displaying the contents of the readme in-app, so this change will cause this paragraph to not show up in there.

Reply:
```suggestion
This configuration strategy is provided as a reference for legacy users. If you are setting up the integration for the first time, consider using the Prometheus-based strategy described in the previous section.
```